slides-02-stattesting.qmd

```diff
@@ -0,0 +1,568 @@
+
+::: {.incremental}
+- Different types:
+  - One-Sample: Comparing our data to a set value
+  - Two-Sample: Comparing two sets of data
+    - Unpaired: Comparing two values for two different groups
+    - Paired: Comparing two values for the same unit
+
+:::
+
+::: {.notes}
+- Unpaired Examples: 
+  - A/B testing
+  - compare test scores for two different classrooms taught by two different instructors
+- Paired Examples: 
+  - compare test scores for each student at the beginning and end of class
+  - water samples from a river before and after a treatment or environmental change
+
+:::
+
+## t-test syntax
+
+Use the `svyttest()` function to compare two proportions or means.
+
+Syntax:
+
+```{r}
+#| label: syntax-ttest-1
+#| eval: false
+
+svyttest(formula,
+         design,
+         ...)
+```
+
+::: {.notes}
+- Similar to `t.test()` function in base R with the formula and the data arguments
+:::
+
+
+## t-test syntax
+
+```{r}
+#| label: syntax-ttest-error
+#| error: true
+
+recs_des %>%
+  svyttest(SummerTempNight - 68 ~ 0, 
+           na.rm = TRUE)
+```
+
+## t-test syntax
+
+Use the `svyttest()` function to compare two proportions or means.
+
+Syntax:
+
+```{r}
+#| label: syntax-ttest-2
+#| eval: false
+#| code-line-numbers: "2"
+
+svyttest(formula,
+         design,
+         ...)
+```
+
+::: {.notes}
+- as `design` argument is the second arugument, dot notation is needed
+:::
+
+# Dot notation {background-color='{{< brand color secondary >}}'}
+
+## Dot notation
+
+Let's walk through an example using the `towny` data.
+
+```{r}
+#| label: dot-notation-0
+
+towny
+
+```
+
+## Dot notation 
+
+Using the `towny` data, let's filter to only cities.
+
+```{r}
+#| label: dot-notation-1
+
+filter(towny, csd_type == "city")
+
+```
+
+## Dot notation
+
+Using the `towny` data, let's filter to only cities.
+
+```{r}
+#| label: dot-notation-2
+#| eval: false
+#| code-line-numbers: "2"
+
+filter(towny, csd_type == "city")
+towny %>% filter(csd_type == "city")
+
+```
+
+```{r}
+#| label: dot-notation-2-run
+#| echo: false
+
+towny %>% filter(csd_type == "city")
+
+```
+
+## Dot notation 
+
+Using the `towny` data, let's filter to only cities.
+
+```{r}
+#| label: dot-notation-3
+#| eval: false
+#| code-line-numbers: "3"
+
+filter(towny, csd_type == "city")
+towny %>% filter(csd_type == "city")
+towny %>% filter(., csd_type == "city")
+```
+
+```{r}
+#| label: dot-notation-3-run
+#| echo: false
+
+towny %>% filter(., csd_type == "city")
+
+```
+
+## Dot notation 
+
+Using the `towny` data, let's filter to only cities.
+
+```{r}
+#| label: dot-notation-4
+#| eval: false
+#| code-line-numbers: "4"
+
+filter(towny, csd_type == "city")
+towny %>% filter(csd_type == "city")
+towny %>% filter(., csd_type == "city")
+towny %>% filter(.data = ., csd_type == "city")
+```
+
+```{r}
+#| label: dot-notation-4-run
+#| echo: false
+
+towny %>% filter(.data = ., csd_type == "city")
+
+```
+
```

Comment:
Might be helpful to add a slide at this point with a working example of dot notation with svyttest for closure